### DISCUSSION TITLE
How to highlight/toggle the image color?

### DISCUSSION THREAD

**Discussion by TheCodeCache** [flet-dev/flet#1033] [Q&A]
Here is an input image - 

<img width="32" alt="image_A" src="https://user-images.githubusercontent.com/26399543/218430929-393fc12c-2296-4b13-af25-f89e6d0427f4.png">

I need to change this to below format - 

<img width="26" alt="image_B" src="https://user-images.githubusercontent.com/26399543/218431255-9ab840bf-f6db-4ee3-9b2c-0c94620d6059.png">

Apologies for the tiny images. 

If we observe the difference b/w the above two images, structure wise they both are same, it's just that their colors are `toggled` especially the center part of the image, which is white in the first image whereas it toggled black in the second.

How to `switch/toggle` color of a given image?
Is there any `attribute` on some flet image control or something similar, which highlights the image or achieve the above effect,?

Kindly advice!

**Comment by FeodorFitsner** [comment]
You can try [ShaderMask](https://flet.dev/docs/controls/shadermask) control to see if it suites your needs.
